build.gradle

```diff
@@ -284,8 +289,12 @@ task unpack(type: Copy) {
     into("build/dependency")
 }
 
+def generatedOutputDir = 'src/generated/java'
+
 clean.doFirst {
     delete("${projectDir}/$generatedOutputDir")
```

Comment:
Documenting our conversation: Since this PR moves the generated code into the build folder, it will automatically be cleared on clean. However, we should keep this code around for people transitioning to these changes because they will still have code in the "/src/generated/java" location. 
TODO: add comment that indicates we can remove lines 292 & 294 after everyone has run the latest version of the code on their local environments.